apps/prairielearn/src/lib/authz-data-lib.ts

```diff
@@ -122,9 +126,7 @@ export const CourseOrInstanceContextDataSchema = z.object({
 });
 export type CourseOrInstanceContextData = z.infer<typeof CourseOrInstanceContextDataSchema>;
 
-export type AuthzDataWithoutEffectiveUser =
-  | ConstructedCourseOrInstanceSuccessContext['authzData']
-  | DangerousSystemAuthzData;
+export type AuthzDataWithoutEffectiveUser = PlainAuthzData | DangerousSystemAuthzData;
 
 export type AuthzDataWithEffectiveUser = PageAuthzData | DangerousSystemAuthzData;
```

Comment:
Note that `PageAuthzData` was actually already branded, so we were already halfway there.